product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -0,0 +1,225 @@
+
+```
+pg_dump --schema-only  -h <old_biganimal_host> -U <old_biganimal_username> -d <old_biganimal_databasename> | psql -h <new_biganimal_host> -U <new_biganimal_username> -d <new_biganimal_databasename>
+```
+
+The `pg_dump --schema-only` command exports the schema (structure) of the existing database without the data. It's then piped into `psql` to import this schema into the new Postgres instance. This prepares the target database with the necessary structure to hold the data.
+
+Finally, log into the new instance and confirm that the database schema migrated correctly. Run `\dt+;` in this example again and you see the database schema migrated to the new instance as expected:
+
+```
+                                         List of relations
+ Schema |       Name       | Type  |   Owner   | Persistence | Access method |  Size   | Description 
+--------+------------------+-------+-----------+-------------+---------------+---------+-------------
+ public | pgbench_accounts | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_branches | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_history  | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_tellers  | table | edb_admin | permanent   | heap          | 0 bytes | 
+```
+
+While the schema looks the same on the new instance, the "Size" column shows 0 bytes of data, confirming that only the schema has been migrated so far.
+
+### Setting up Publication 
+
+To set up the publication, log into the old BigAnimal instance and create a publication:
+
+```sql
+CREATE PUBLICATION <pub_name>;
+```
+
+In this example:
+
+
+```sql
+CREATE PUBLICATION v12_pub;
+```
+
+The expected output is `CREATE PUBLICATION`.
+
+Next, configure the publication to include the specific tables you want replicated on the new instance:
+
+```sql
+ALTER PUBLICATION <pub_name> ADD TABLE <table_name>;
+```
+
+Which in the current example is:
+
+```sql
+ALTER PUBLICATION v12_pub ADD TABLE pgbench_accounts;
+ALTER PUBLICATION v12_pub ADD TABLE pgbench_branches;
+ALTER PUBLICATION v12_pub ADD TABLE pgbench_history;
+ALTER PUBLICATION v12_pub ADD TABLE pgbench_tellers;
+```
+
+Each of these alterations produces `ALTER PUBLICATION` output upon success. 
+
+### Creating the Logical Replication Slot
+
+Then, on the older versioned instance, create a replication slot using the 'pgoutput' plugin:
+
+```sql
+SELECT pg_create_logical_replication_slot('<slot_name>','pgoutput');
+```
+
+In the current example:
+
+```sql
+SELECT pg_create_logical_replication_slot('v12_pub','pgoutput');
+```
+
+If successful, the expected output is something like the following:
+
+```
+ pg_create_logical_replication_slot 
+------------------------------------
+ (v12_pub,0/AC003330)
+```
+
+This slot tracks changes to the published tables on the old instance to ensure they can be replicated to the subscriber on the new instance without losing any data.
```

Comment:
The replication slot tracks changes to the published tables from the source instance and replicates changes to the subscriber on the target instance.

Reply:
👍